src/azure-cli/azure/cli/command_modules/databoxedge/generated/_help.py

```diff
@@ -0,0 +1,332 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['databoxedge device'] = """
+    type: group
+    short-summary: Manage device with databoxedge
+"""
+
+helps['databoxedge device list'] = """
```

Comment:
use active voice for all command description

Reply:
I have overwrite all helps under manual folder